core/common/src/main/java/alluxio/Constants.java

```diff
@@ -118,7 +118,8 @@
   public static final String KEY_VALUE_WORKER_CLIENT_SERVICE_NAME = "KeyValueWorkerClient";
 
   public static final String REST_API_PREFIX = "v1/api";
-  public static final String LOGGER_TYPE = PropertyKey.LOGGER_TYPE;
+  public static final String LOGGER_TYPE = PropertyKey.LOGGER_TYPE.toString();
```

Comment:
If we store the string values in PropertyKey.Constants it could be `PropertyKey.Constants.LOGGER_TYPE`. Alternately, could we create the Logger in one place and use it everywhere, e.g. 
```java 
protected static final Logger LOG = CommonUtils.LOG;
```